### PR TITLE
suel indexes

### PR DESCRIPTION
**Pull request by AFoletti**

### CODE REVIEW

**Comment by AFoletti** [comment]
that's ready to be merged once reviewed

File: conf/uvek.conf.part [hunk 1]
```diff
@@ -765,3 +902,68 @@ index ch_bfe_windenergieanlagen : ch_astra_ivs-nat
...
+    path = /var/lib/sphinxsearch/data/index/ch_bfe_sachplan-uebertragungsleitungen_anhorung_4
+}
+
+index ch_bfe_sachplan-uebertragungsleitungen_anhorung 
+{
+    type = distributed
+    local = ch_bfe_sachplan-uebertragungsleitungen_anhorung_1
+    local = ch_bfe_sachplan-uebertragungsleitungen_anhorung_2
+	local = ch_bfe_sachplan-uebertragungsleitungen_anhorung_3
+	local = ch_bfe_sachplan-uebertragungsleitungen_anhorung_4
```
Comment (gjn) [inline conf/uvek.conf.part]: ident

File: conf/uvek.conf.part [hunk 2]
```diff
@@ -765,3 +902,68 @@ index ch_bfe_windenergieanlagen : ch_astra_ivs-nat
...
+}
+
+index ch_bfe_sachplan-uebertragungsleitungen_kraft 
+{
+    type = distributed
+    local = ch_bfe_sachplan-uebertragungsleitungen_kraft_1
+    local = ch_bfe_sachplan-uebertragungsleitungen_kraft_2
+	local = ch_bfe_sachplan-uebertragungsleitungen_kraft_3
+    local = ch_bfe_sachplan-uebertragungsleitungen_kraft_4
+}
```
Comment (gjn) [inline conf/uvek.conf.part]: strange character here...

File: conf/uvek.conf.part [hunk 3]
```diff
@@ -765,3 +902,68 @@ index ch_bfe_windenergieanlagen : ch_astra_ivs-nat
...
+    source = src_ch_bfe_sachplan-uebertragungsleitungen_kraft_4
+    path = /var/lib/sphinxsearch/data/index/ch_bfe_sachplan-uebertragungsleitungen_kraft_4
+}
+
+index ch_bfe_sachplan-uebertragungsleitungen_kraft 
+{
+    type = distributed
+    local = ch_bfe_sachplan-uebertragungsleitungen_kraft_1
+    local = ch_bfe_sachplan-uebertragungsleitungen_kraft_2
+	local = ch_bfe_sachplan-uebertragungsleitungen_kraft_3
```
Comment (gjn) [inline conf/uvek.conf.part]: ident

**Pull request merged**